src/test/scala/chiselTests/interface/InterfaceSpec.scala

```diff
@@ -167,6 +173,52 @@ class InterfaceSpec extends AnyFunSpec with Matchers {
       Drivers.link(dir, "compile-0/Foo.sv")
 
     }
+  }
+
+  describe("Basic behavior of properties") {
+
+    // A container of the types of properties that are on this Interface.
+    case class SomeProperties(a: Int, b: String, c: Int => Int)
```

Comment:
Both `b` and `c` are not used at all here and could be removed. I included them to show that arbitrary things can be properties, including functions.